### PR TITLE
Bug/issue enhanced  use latest sprint

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
**Are you a current Fivetran customer?** 

Fivetran created PR.

**What change(s) does this PR introduce?** 

## 🐞 Bug Fixes
Updated logic model `int_jira__issue_sprint` to adjust how current sprint is determined. It now uses the newest `started_at` date of the sprint instead of the `updated_at` date of the issue.

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature #71 
- [ ] No 

**How did you test the PR changes?** 

 

- [x] Buildkite  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:seedling:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+## 🚨 Breaking Change
+For model `jira__issue_enhanced`, update column names `sprint_id` and `sprint_name` to `current_sprint_id` and `current_sprint_name`, respectively, to confirm the record is for the current sprint.
+
+## 🐞 Bug Fix
+Updated logic for model `int_jira__issue_sprint` to adjust how current sprint is determined. It now uses the newest `started_at` date of the sprint instead of the `updated_at` date.
 # dbt_jira v0.9.0
 ## 🚨 Breaking Changes 🚨
 
-- The default schema for the source tables are now built within a schema titled (`<target_schema>` + `_jira_source`) in your destination. The previous default schema was (`<target_schema>` + `_stg_jira`) for source. This may be overwritten if desired. ([#63](https://github.com/fivetran/dbt_jira/pull/63))
+- The default schema for the source tables are now built within a schema titled (`<target_schema>` + `_jira_source`) in your destination. The previous default schema was (`<target_schema>` + `_stg_jira`) for source. This may be overwritten if de`sired. ([#63](https://github.com/fivetran/dbt_jira/pull/63))
```
Comment (fivetran-reneeli) [inline CHANGELOG.md:11]: a lil typo!

**Review by fivetran-reneeli — CHANGES_REQUESTED** [review]
@fivetran-catfritz  looks good, just had some minor edits!

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-catfritz) [reply CHANGELOG.md]: Fixed!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,14 @@
+# dbt_jira v0.10.0
+[PR #76](https://github.com/fivetran/dbt_jira/pull/76) includes the following changes:
+## 🚨 Breaking Change
```
[the remark this answers is not in the document]
Comment (fivetran-catfritz) [reply CHANGELOG.md]: Updated!

**Comment by fivetran-catfritz** [comment]
@fivetran-reneeli I have updated and ready for your review! Thanks!

**Review by fivetran-reneeli — APPROVED** [review]
Looks good!

**Pull request merged**